docs/help/contributor/style-guide/documenting-code.md

```diff
@@ -258,3 +258,16 @@ Apply code formatting only to special-purpose text:
     <some-code>
     ```
     ````
+
+
+## Documentation of flags
+> Always use the short version of flags used in a command than the extended version, i.e., use ```-n``` rather than ```--namespace```
```

Comment:
```suggestion
> Always use the short version of flags used in a command rather than the extended version. For example, use `-n` rather than `--namespace`.
```